combat-trainer.lic

```diff
@@ -3869,7 +3894,7 @@ class GameState
     options -= held_types if weapon_skill.eql?('Bow') || (weapon_skill.eql?('Crossbow') && !@using_light_crossbow)
     options -= thrown_types if (weapon_skill.eql?('Bow') && @left_hand_free)
     echo "determine_aiming_skill::options: #{options}" if $debug_mode_ct
-    sort_by_rate_then_rank(options).first
+    options.min_by { |skill| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept